cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -228,10 +235,10 @@ func (v *volumeAPIOpsV1alpha1) list() (*v1alpha1.CASVolumeList, error) {
 
 	cvols, err := vOps.List()
 	if err != nil {
-		glog.Errorf("failed to list cas template based volumes: error '%s'", err.Error())
+		glog.Errorf("failed to list cas template based volumes at namespaces '%s': error '%s'", vols.Namespace, err.Error())
 		return nil, CodedError(500, err.Error())
 	}
 
-	glog.Infof("cas template based volumes were listed successfully")
+	glog.Infof("cas template based volumes were listed successfully: namespaces '%s'", vols.Namespace)
```

Comment:
Will the value of vols.Namespace be populated in the following scenarios:
1) ALL namespace.
2) DEFAULT namespace.